src/functions-templates/js/fauna-crud/create-schema.js

```diff
@@ -14,25 +14,23 @@ const createFaunaDB = function () {
   })
 
   /* Based on your requirements, change the schema here */
-  return client
-    .query(query.CreateCollection({ name: 'items' }))
-    .then(() => {
-      console.log('Created items class')
-      return client.query(
-        query.CreateIndex({
-          name: 'all_items',
-          source: query.Collection('items'),
-          active: true,
-        }),
-      )
-    })
+  try {
+    await client.query(query.CreateCollection({ name: 'items' }))
 
-    .catch((error) => {
-      if (error.requestResult.statusCode === 400 && error.message === 'instance not unique') {
-        console.log('DB already exists')
-      }
-      throw error
-    })
+    console.log('Created items class')
+    return client.query(
```

Comment:
```suggestion
    return await client.query(
```

For proper error handling.